update-examples-maven.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+
+function generate_maven_pom_xml() {
+    local VERSION=$1
+    local SRC_DIR=$2
+    local CUR_DIR=$3
+
+    echo 'Generate maven pom.xml for examples ...'
+    # clone shalk/armeria-tom4j to `.mvn/armeria-tom4` or pull if exists
+    local MVN_TOOL_DIR=".mvn/armeria-tom4j"
+    if [[ -d $MVN_TOOL_DIR ]]
+    then
+      pushd $MVN_TOOL_DIR && git pull && popd
+    else
+      git clone https://github.com/shalk/armeria-tom4j $MVN_TOOL_DIR
```

Comment:
armeria-toml looks very nice. I'd like to use armeria-tom4j to create `build.gradle` files as well. Are you also interested in that? 😉

Reply:
I think it is good to use project to generate gradle file.

Just need rewrite some bash in java, maybe easy to maintain. But gradle file is not structful.

We can discuss this in new issue.

Reply:
@shalk The [armeria-tom4j](https://github.com/shalk/armeria-tom4j) looks very useful. Thanks!
It seems like the project is created only for this armeria-examples repository. How about just putting all the source codes into this repository to manage it together?

Reply:
@minwoox that is a good idea. I merged the project in.